examples/pecos-multi-node-AWS-batch-CDK/pecos-multi-node-batch-job/source.bat

```diff
@@ -0,0 +1,13 @@
+@echo off
+
+rem The sole purpose of this script is to make the command
+rem
+rem     source .venv/bin/activate
+rem
+rem (which activates a Python virtualenv on Linux or Mac OS X) work on Windows.
+rem On Windows, this command just runs this batch file (the argument is ignored).
+rem
+rem Now we don't need to document a Windows command for activating a virtualenv.
+
+echo Executing .venv\Scripts\activate.bat for you
+.venv\Scripts\activate.bat
```

Comment:
Delete this file since `.venv` does not exist in this PR.

Reply:
Sounds good, will do it soon.